runtime/doc/syntax.txt

```diff
@@ -5481,14 +5481,14 @@ memory Vim will consume.
 Only highlighting typedefs, unions and structs can be done too.  For this you
 must use Universal Ctags (https://ctags.io) or Exuberant ctags.
 
-Put these lines in your Makefile:
+Put these lines in your Makefile: >
```

Comment:
closing marker -- and arguably, _if_ we add new markers to Vim help files, we might as well annotate them (although _here_ it doesn't make much sense until we ship a makefile parser)